Update docs welcome page (#228)
nebari-dev/nebari-docs: docs/docs/welcome.mdx

slug: /welcome
---

Nebari is an open source data science platform, developed for collaboration and scalability.

> It aims to provide a robust infrastructure that can be set up (deployed) quickly.

Reviewer: ```suggestion
It aims to provide a robust infrastructure that can quickly be set up (deployed).
```

![Nebari overview sequence showing deploying from yaml to cloud resources to the instance spinup selection](/img/welcome/nebari_overview_sequence.png)

## Get Started

New to Nebari? Start here to set up your own Nebari instance.

> [Learn how to install and deploy Nebari from scratch →](./get-started/installing-nebari)

> Refer to [CLI commands in the Quick start →](./get-started/quickstart)

<!-- TODO: All a link to Nebari 101 when it's ready. -->

> [↳ Read all pages][get-started]

Reviewer: ```suggestion
[↳ Read all the getting started resources][get-started]
```

[↳ Read all tutorials][tutorials]

## How-to guides

Learn to perform specific tasks in Nebari, with step-by-step instructions.

> If you're an administrator, read [how-to configure Keycloack for all users →][configure-keycloak]

> If you're a data scientist, [learn how-to use Visual Studio code for your work →][using-vscode]

[↳ Read all guides][how-tos]

<!-- TODO: Add conceptual guides when ready

## Conceptual guides

> TBD

[↳ Read all guides](./) -->

## Community

Nebari is a community-led open source project, learn how you can get involved!

> [Facing issues with Nebari? Get support on GitHub discussions or the issue tracker →](./community#getting-support)

> [Learn how you can contribute to Nebari, beyond code and pull-requests →](./community#how-to-contribute)

[↳ Read all community resources][community]

## References

<!-- TODO: Add API doc when ready -->

Understand technical details of Nebari's architecture and how it works.

> [Release notes →][release-notes]

[↳ Read all references][references]

## Learning paths

Nebari's individual documentation page have specific purpose and an intended audience.

> You can be an administrator configuring resourcesfor your team, or a data scientist looking to create dashboards.

Reviewer: ```suggestion
You can be an administrator configuring resources for your team or a data scientist looking to create dashboards (or you might alternate among roles over time).
```

The following tabs provide a quick guide for the different paths you can take to navigate Nebari's documentation.

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

<Tabs>

>   <TabItem value="data-scientist" label="Data scientist" default>

Reviewer: What about renaming to data practitioner here too?

>   <TabItem value="data-scientist" label="Data scientist" default>

Author: ```suggestion
  <TabItem value="data-practitioner" label="Data practitioner" default>
```

>   If you are a data professional using Nebari for your day-to-day work like data analysis, visualization, and more, the following path and pages will be useful for you.

Reviewer: ```suggestion
  If you are a data practitioner using Nebari for your day-to-day work, like data analysis, visualization, and more, the following path and pages will be useful for you.
```

Let's remove replace with practitioner

  🔸 [Log-in to your team's Nebari instance, launch your JupyterLab server](./how-tos/login-keycloak), and use the IPython/Jupyter notebooks.

  🔸 Learn to [create new environments (kernels) using conda-store](./tutorials/creating-new-environments) if you need specific libraries.

  🔸 If you need to scale your work, [leverage Dask and Dask Gateway for parallel and distributed computing within Nebari][using-dask].

  🔸 Learn to [create and share interactive data visualization dashboards with your team][create-dashboards].

>   🔸 If you need to run notebooks or scripts regularly (cronjobs), [you can read how to use `kbatch`](./how-tos/kbatch-howto).

Reviewer: ```suggestion
  🔸 If you need to run notebooks or scripts regularly (such as ronjobs), [you can read how to use `kbatch`](./how-tos/kbatch-howto).
```

  🔸 You can also [consider Visual Studio Code][using-vscode] if that is more convenient for your work.

  </TabItem>

  <TabItem value="administrator" label="Administrator">

>   If you are setting up Nebari and managing the platform for your organization, the following steps and pages will be helpful.

Reviewer: ```suggestion
  If you are setting up Nebari and managing the platform for your organization or team, the following steps and pages will be helpful.
```
